classes/Tab.php

```diff
@@ -150,7 +150,10 @@ public static function initAccess($id_tab, Context $context = null)
 
 	public function delete()
 	{
-		if (Db::getInstance()->execute('DELETE FROM '._DB_PREFIX_.'access WHERE `id_tab` = '.(int)$this->id) && parent::delete())
+	 	if (Db::getInstance()->execute('DELETE FROM '._DB_PREFIX_.'access WHERE `id_tab` = '.(int)$this->id) &&
+			Db::getInstance()->execute('DELETE FROM '._DB_PREFIX_.'tab_lang WHERE `id_tab` = '.(int)$this->id) &&
+			parent::delete())
+		{
```

Comment:
Hi,

Thank you for your PR. I believe your are adding an additional bracket here.

Regards